AzureCalling/External/Calling/BottomDrawerCellViewModel.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+struct BottomDrawerCellViewModel {
+    let avatar: UIImage
+    let title: String
+    var enabled: Bool
+}
+
+extension BottomDrawerCellViewModel {
+    func convertToAudioDeviceDataModel() -> AudioDeviceDataModel {
```

Comment:
Do we need to convert the viewmodel back to data model?

Reply:
My refactor should address this